doc/src/docutils/tools/doxygen_xml2qbk/doxygen_xml_parser.hpp

```diff
@@ -458,9 +442,58 @@ static void parse_element(rapidxml::xml_node<>* node, configuration const& confi
             parse_parameter(node->first_node(), config, p);
             el.parameters.push_back(p);
         }
+        else if (boost::contains(full, ".qbk."))
+        {
```

Comment:
The code below replaces the block above but is a bit safer in terms of full paths.
Also it should not handle sub nodes in some cases, to avoid duplicates.